third_party/terraform/website/docs/r/dataproc_cluster.html.markdown

```diff
@@ -431,6 +431,16 @@ cluster_config {
    a cluster. For a list of valid properties please see
   [Cluster properties](https://cloud.google.com/dataproc/docs/concepts/cluster-properties)
 
+* `optional_components` - (Optional) The set of optional components to activate on the cluster. Possible values include:
+    "ANACONDA"
```

Comment:
This doesn't actually render one per line (https://github.com/modular-magician/terraform-provider-google/compare/auto-pr-3294-old..auto-pr-3294?short_path=2f9b3da#diff-2f9b3da2dac59a097b571cb7c644eabf, rich diff)